package.json

```diff
@@ -36,8 +36,11 @@
     "async": "^1.5.2",
     "buffer-equal": "^1.0.0",
     "cesium": "^1.17.0",
+    "clone": "^1.0.2",
```

Comment:
Could we just use `Cesium.clone` instead? - https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/clone.js